t.css

```
@@ -0,0 +1,5 @@
+a {}
```

Expected empty line before at-rule.

```suggestion

@media {}```